turnserver/internal/utils/response.go

```diff
@@ -0,0 +1,35 @@
+package utils
```

Comment:
these functions should be in the src/controller pkg.. For that matter, could be added to controller.go